libr/util/str_trim.c

```diff
@@ -82,6 +82,10 @@ R_API char *r_str_trim_dup(const char *str) {
 
 R_API void r_str_trim(char *str) {
```

Comment:
this seems just a dup of `r_str_trim_head_tail`

Reply:
yep indeed. this is fixed now